sites/jeromefitzgerald.com/src/components/Notion/Blocks/Image.tsx

```diff
@@ -216,5 +215,6 @@ async function ImageImpl({
   )
 }
 
+export { Image as NextImage }
```

Comment:
🥴 This makes my head hurt a bit.

Reply:
Well. Still hurts, but okay. `Image.client` does not seem as bad as that is more of a wrapper.